Podfile

```diff
@@ -0,0 +1,9 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '11.0'
+
+workspace 'StreamChatIntegration'
+
+target 'StreamChatIntegration-CocoaPods' do
+  project 'StreamChatIntegration-CocoaPods.xcodeproj'
+  pod 'StreamChatUI'
```

Comment:
Are we going to be using this for PR merges into main as well? If that's the case, we would need to pass the branch

Reply:
For now we can keep it this way. If we will need to point the pods towards some specific branch during our development time we can do so, once the setup for binary frameworks will be in place